Formula/g/gurk-rs.rb

```diff
@@ -0,0 +1,18 @@
+class GurkRs < Formula
+  desc "Signal Messenger client for terminal"
+  homepage "https://github.com/boxdot/gurk-rs"
+  url "https://github.com/boxdot/gurk-rs/archive/refs/tags/v0.5.2.tar.gz"
+  sha256 "a8c190e081d9905ea6e71d38d56cb8f19596999a20cb9584a552ce46e6f88bed"
+  license "AGPL-3.0-or-later"
+
+  depends_on "protobuf" => :build
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/gurk --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I can also test the help page, but gurk doesn't have much more to test, it's a GUI app, but in the terminal and you have to connect your phone before messaging anyone.